revocation/revocation.go

```diff
@@ -104,6 +112,10 @@ type Options struct {
 	// values are CodeSigning and Timestamping. Default value is CodeSigning.
 	// OPTIONAL.
 	CertChainPurpose purpose.Purpose
+
+	// CRLCache is the cache client used to store the CRL. if not provided,
+	// a default in-memory cache will be used.
```

Comment:
Should we consider no cache if it is set to nil?

Reply:
Of course, we need to mention the file cache and memory cache in the documentation.